Flutter/lib/CreateSettle.dart

```diff
@@ -1,34 +1,26 @@
-import 'package:flutter/foundation.dart';
-import 'package:flutter/material.dart';
-import 'package:flutter/services.dart';
-import 'package:flutter_spinkit/flutter_spinkit.dart';
-import 'package:animated_dialog_box/animated_dialog_box.dart';
-import 'package:google_fonts/google_fonts.dart';
-import 'package:share/share.dart';
-import 'LobbyScreen.dart';
-import 'Server.dart';
-import 'Settle.dart';
-import 'app_localizations.dart';
+import 'import_all.dart';
```

Comment:
Again, importing everything seems unnecessary and bad practice

Reply:
Explained in a previous request.